selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -596,4 +597,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return device.hashCode();
   }
+
+  public String getDeviceType() {
+    return deviceType;
```

Comment:
Thanks @operatiay 
For real device, the deviceType is indicated by the hardware model. This hardware model is from the device itself and will be available after an initialization while adding the device to the device store.